tests/actions/test_models.py

```diff
@@ -1,6 +1,4 @@
-from datetime import datetime
-
-from django.utils import timezone
+from datetime import datetime, timezone
```

Comment:
Minor nit: in other projects I usually import the datetime timezone with an alias (e.g.`from datetime import timezone as dt_timezone`), because it's quite common to also need to use things from `django.utils.timezone` (`timezone.now`, particularly).

Reply:
Thanks again. My aim was to touch the codebase as little as possible, so I'm not going to use an alias now. However, I may use an alias later.